src/server/system_services/account_server.js

```diff
@@ -719,6 +722,18 @@ async function add_external_connection(req) {
     info.secret_key = system_store.master_key_manager.encrypt_sensitive_string_with_master_key_id(
         req.rpc_params.secret, req.account.master_key_id._id);
 
+    if (req.rpc_params.azure_log_access_keys) {
+        info.azure_log_access_keys = {
+            azure_tenant_id: req.rpc_params.azure_log_access_keys?.azure_tenant_id,
```

Comment:
you already checked that req.rpc_params.azure_log_access_keys defined, so no need to add the question mark  on every line
for instance - 
req.rpc_params.azure_log_access_keys?.azure_tenant_id,